src/dispatch/FetchHttpHandler.ts

```diff
@@ -88,37 +88,38 @@ export class FetchHttpHandler implements HttpHandler {
 
         const fetchRequest = new Request(url, requestOptions);
         const raceOfPromises = [
-            this.fetchFunction!.apply(window, [fetchRequest]).then(
-                (response) => {
-                    const fetchHeaders: any = response.headers;
-                    const transformedHeaders: HeaderBag = {};
-
-                    for (const pair of fetchHeaders.entries() as string[][]) {
-                        transformedHeaders[pair[0]] = pair[1];
-                    }
-
-                    const hasReadableStream = response.body !== undefined;
-
-                    // Return the response with buffered body
-                    if (!hasReadableStream) {
-                        return response.blob().then((body) => ({
-                            response: new HttpResponse({
-                                headers: transformedHeaders,
-                                statusCode: response.status,
-                                body
-                            })
-                        }));
-                    }
-                    // Return the response with streaming body
-                    return {
+            this.fetchFunction!.apply(window, [
+                fetchRequest,
+                { keepalive: true }
```

Comment:
This line is the only change. Unfortunately, the linter forced me to make all of these formatting changes, and I could not exclude them.